internal/config/config.go

```diff
@@ -67,6 +67,9 @@ type Config struct {
 	// description: |
 	//   Name of the cluster.
 	Name string `yaml:"name" validate:"valid_name,required"`
+	// description : |
+	//	 Custom tags to be applied to created cloud resources.
```

Comment:
```suggestion
	//   Custom tags to be applied to created cloud resources.
```
I think there is a tab character in here that breaks comment generation

As a nit, I would also prefer to have this moved down to just before `Provider ProviderConfig`

Reply:
Unfortunately, the tab is not what messes up comment generation. I'll try to find a fix.

Reply:
Removing the tab fixed it for me